src/bin/collapse-dtrace.rs

```diff
@@ -26,13 +26,22 @@ struct Opt {
     #[structopt(short = "v", long = "verbose", parse(from_occurrences))]
     verbose: usize,
 
+    /// Include offsets
+    #[structopt(short = "o", long = "includeoffset")]
```

Comment:
Let's not use `-o` for this, since `-o` commonly means "output file". Instead, let's make the `long` shorter. How about `--offsets`?

Reply:
👍

Reply:
hmm, writing it, no offset really doesn't tell muych about what it does, how about `-i`? and keep it the same name as upstream?

Reply:
`-i` usually denotes "input file" :)
I think `--offsets` is pretty reasonable given the context -- what makes you think it isn't? Keep in mind that users will also see the help text (which we should probably expand a little) when they run `-h` or `--help`. Maybe make the doc-comment for this option:
```rust
/// Include function offsets for all but the last location in the call chain.
```

Reply:
It seems reasonable for a flag to say what it does, a flag named `--offset` usually lets you specify an offset to something, not prevent the program to skip something. `--includeoffset` does what it says, it includes the offset. The argument for the help text can as well be made for `-i` or `-o`; That all said I don't think that flag was ever used upstream, how about just not giving it a short?

Reply:
also `-i` on grep is "ignore case" on sed it's "add extension" for ls it's "find serial number"

Reply:
Ah, no, `--offsets`. I think the plural is important. That said, I'm okay with just keeping the long version and having it be `--includeoffset`.

The reason I'm pushing back on `-i` and `-o` is that I think it's likely we'll want to add arguments specifically for input and output files in the future, and those would be the obvious candidates for those arguments :)

Reply:
👍 `--includeoffset` only seems like the best compromise then!